<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewTask</class>
 <widget class="QDialog" name="ViewTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查看当前任务</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>0</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonGetSubTask">
          <property name="text">
           <string>提取子任务结果</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonQuit">
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>80</number>
      </attribute>
      <column>
       <property name="text">
        <string>任务名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>任务描述</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务状态</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务负责人</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务开始时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务结束时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务实际完成日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>子任务中间数据结果</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
